### PR TITLE
164069173 Setup testing environment

### PR DESCRIPTION
**Pull request by joelethan**
#### What does this PR do?
- This PR sets up the testing environment for the react project
#### Description of Task to be completed?
- Install packages and dependencies for testing
- Write a test that the application renders without crushing
#### How should this be manually tested?
- Clone the [repo](https://github.com/andela/ah-aquaman-frontend) and cd into `ah-aquaman-frontend`
- checkout the branch `ch-setup-test-env-164069173`
- Run the commands below to install packages and run tests
    -  npm install
    -  npm test
#### What are the relevant pivotal tracker stories?
- [#164069173](https://www.pivotaltracker.com/story/show/164069173)

#### Screenshots (if appropriate)
- expected output
Screenshot 2019-04-02 at 17 40 42

### CODE REVIEW

**Review by BagzieGracious — APPROVED** [review]
Thank you @joelethan, for your good work

**Review by Peace-Apple — APPROVED** [review]
Good work @joelethan

**Review by wasibani-roy — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "react-scripts test --watchAll",
```
Comment (AmosWels) [inline package.json]: Can remove the `--watchAll` so testers can select to use it when they need to

**Review by AmosWels — CHANGES_REQUESTED** [review]
Goodwork @joelethan , please create a `__tests__` folder in the src director and add the `App.test.js` to that folder.

**Comment by Peace-Apple** [comment]
Good work @joelethan, please rebase with the develop branch to resolve the conflicts your branch is having.

**Review by AmosWels — APPROVED** [review]

**Pull request merged**